lib/open_api_spex.ex

```diff
@@ -205,6 +205,8 @@ defmodule OpenApiSpex do
   @doc """
   Raises compile time errors for improperly defined schemas.
   """
+  @spec error!(atom(), %Schema{}, keyword()) :: no_return()
+  @spec error!(atom(), %Schema{}) :: no_return()
```

Comment:
The Schema struct has a type: `Schema.t()`. Want to use that instead?

Reply:
👍 totally, great catch.